models.py

```diff
@@ -61,24 +84,24 @@ class Chapter(db.Model):
     __tablename__ = 'ChapterData'
 
     def serialize(self):
+        bookNumber = Book.get_number_from_id(self.arabicBookID)
         return {
+            'bookNumber': bookNumber,
             'chapterNumber': str(self.babID),
```

Comment:
I would rather not expose `babID`, plus we have `chapterNumber` in the language-specific blocks. Is this one required?